pravega-k8-log-collection/pravega-k8s-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Use `#!/usr/bin/env bash`. In some systems `bash` is not located in `/bin/bash`, the suggested shebang will try to find `bash` in the path.

Reply:
+1.
Sure will update.